src/etl/internal/adapter/repository_test.py

```
@@ -0,0 +1,81 @@
+        MessageMetadata(sender_id=10, chat_id=505,
+                        text="Добавил новые иконки в репозиторий", attached_files=[]),
+        MessageMetadata(sender_id=12, chat_id=505,
+                        text="Does anyone have the link to the dev server?",
+                        attached_files=[])
+    ]
+    start = time.perf_counter()
+    await repo.save_batch(extra_messages)
+    end = time.perf_counter()
+    print(f"✅ Пакет успешно загружен. Время выполнения - {end-start}")
+
+    query = "проблемы с доступом и логином"
```

Ruff (0.15.10)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)